source/read-write-concern/tests/README.rst

```diff
@@ -45,22 +45,43 @@ the spec is final.
 Format
 ------
 
+Connection String
+~~~~~~~~~~~~~~~~~
+
+These tests are designed to excersize the connection string parsing related
+to read concern and write concern.
+
 Each YAML file contains an object with a single ``tests`` key. This key is an
 array of test case objects, each of which have the following keys:
```

Comment:
may.
"each of which may have the following keys".
Not all of them are used all the time